infrastructure/templates/public-api/main.bicep

```diff
@@ -360,15 +379,38 @@ module appGatewayModule 'application/shared/appGateway.bicep' = if (deployContai
   }
 }
 
-module dataProcessorModule 'application/public-api/publicApiDataProcessor.bicep' = {
+var adminSubnetFirewallRule = {
+  name: 'Admin App Service subnet range'
+  cidr: vNetModule.outputs.adminAppServiceSubnetCidr
+  tag: 'Default'
+  priority: 100
+}
```

Comment:
Does this need to be a separate variable? Would be tempted to inline it.

If it does need to be a variable, perhaps it should be grouped with `maintenanceFirewallRules` so they're next to each other.

Reply:
Rather than retain it as a variable, I've just made it inline within the `functionAppFirewallRules` property:

```
    functionAppFirewallRules: union([
      {
        name: 'Admin App Service subnet range'
        cidr: vNetModule.outputs.adminAppServiceSubnetCidr
        tag: 'Default'
        priority: 100
      }
      // TODO EES-5446 - remove service tag whitelisting when runner scale set IP range reinstated 
      {
        cidr: 'AzureCloud'
        tag: 'ServiceTag'
        priority: 101
        name: 'AzureCloud'
      }
```